modules/KalturaSupport/resources/mw.ClosedCaptions.js

```diff
@@ -494,10 +498,13 @@
 			// Return a "textSource" object:
 			return new mw.TextSource( embedSource, _this.embedPlayer );
 		},
-		forceLoadLanguage: function(){
-			var lang = this.getConfig('forceLoadLanguage');
+		forceLoadLanguage: function(language){
+			var lang = this.getConfig('forceLoadLanguage') ? this.getConfig('forceLoadLanguage') : language;
```

Comment:
@eitanavgil why not just use the already existing `'forceLoadLanguage'` option to accomplish this?